syntax="proto2";

package aap_protobuf.service.control.message;

message PingResponse {
    required int64 timestamp = 1;
    optional bytes data = 2;
}
